org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/EclipseSbtBuildManager.scala

```diff
@@ -179,10 +184,11 @@ class EclipseSbtBuildManager(val project: IScalaProject, settings0: Settings) ex
   }
 
   // take by-name argument because we need incOptions only when we have a cache miss
-  private[zinc] def latestAnalysis(incOptions: => IncOptions): Analysis =
+  def getLatestAnalysis(incOptions: => IncOptions): Analysis =
```

Comment:
ok, but I used it just as a verb not bean accessor. but if the 'get' is somehow restricted I'm going to remove it